src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/LLVMObjectWriter.cs

```diff
@@ -814,10 +814,17 @@ private void GetCodeForExternMethodAccessor(ExternMethodAccessorNode node)
             LLVMValueRef externFunc = externFuncModule.AddFunction(externFuncName, externFuncType);
 
             // Add import attributes if specified.
-            if (_compilation.ConfigurableWasmImportPolicy.TryGetWasmModule(externFuncName, out string wasmModuleName))
+            // Copy the prefix from WasmConfigurablePInvokePolicy here to avoid making the policy classes public.
```

Comment:
It would be better to consolidate this logic in the policy.

You can add a virtual `bool GenerateWasmImportCall(MethodDesc method, out string name, out string moduleName)` to `PInvokeILEmitterConfiguration`, and expose it through `PInvokeILProvider` as, e. g., `GetWasmImportCallInfo`